### ISSUE TITLE
Authenticator is not refreshing certificate after expiration

### ISSUE THREAD

**Issue by izgeri** [cyberark/conjur-authn-k8s-client#16]
Some users are reporting that authenticator sidecars deployed for longer than three days are getting 401 errors on authenticate or `INVALID TOKEN`, apparently because the certificate that is initially injected into the pod is not refreshed as expected and thus the authentication stops working after it expires.

Steps to reproduce:
- Configure an authenticator sidecar to retrieve secrets from Conjur, and deploy the sidecar together with an application to Kubernetes
- Wait 3 days for the certificate to expire
- Verify that the application is no longer able to authenticate with Conjur due to expired certificate

**Comment by dsbyrne** [comment]
Someone could verify this in a more timely manner by changing the expiry time here:
https://github.com/cyberark/conjur/blob/master/app/domain/util/open_ssl/ca.rb#L36

**Comment by jvanderhoof** [comment]
From the customer:

Now that we have had some apps deployed for a awhile we noticed that we were getting the error : INVALID TOKEN
    
It appears that the TLS cert being injected back into the authenticator is not getting refreshed and stops working after it expires.   This is a huge problem for us.

We are asking teams to run the authenticator as a sidecar and pull their credentials each time they are needed (in order to catch password rotations).   If the sidecar is going to fail after a couple of days and need to be restarted we will not be able to put Conjur into production.
    
I see a new version of the authenticator was published today 0.12.0.   Any chance this is fixed in this version?
        
I found in the conjur UI this error:
```
conjur/authn-k8s/ocp/apps/grp-inf-csi-conjur-ref/*/*<https://urldefense.proofpoint.com/v2/url?u=https-3A__conjur-2Dsat-2Dprod_ui_hosts_conjur-252Fauthn-2Dk8s-252Focp-252Fapps-252Fgrp-2Dinf-2Dcsi-2Dconjur-2Dref-252F-2A-252F-2A&d=DwIGaQ&c=E55fojPA83XrPGfndbiaQQ&r=AxSHpnf_u7R5KGUmB-VPZyxGaeV51dJ2XxlqDXdr21Y&m=bdR3gJ9kcUs7LpVdSrLBmhTJbXof82WY23KCxpWkFzk&s=DUl8hu3VpCsnSQMKc16cKF_IxuiM6WAQflDPoz_KPzg&e=> failed to authenticate with authenticator authn-k8s service <customer>:webservice:conjur/authn-k8s/ocp: Client certificate expired
```
    
I found this in conjur-follower log:
```
2019-03-06T15:52:09.251+00:00 conjur-follower-2-9rg8m conjur-possum: I, [2019-03-06T15:52:09.011387 #900] INFO -- : Started POST "/authn-k8s/ocp/<customer>/host%2Fconjur%2Fauthn-k8s%2Focp%2Fapps%2Fgrp-inf-csi-conjur-ref%2F%2A%2F%2A/authenticate" for 10.36.36.1 at 2019-03-06 15:52:09 +0000

2019-03-06T15:52:09.251+00:00 conjur-follower-2-9rg8m conjur-possum: I, [2019-03-06T15:52:09.012208 #900] INFO -- : Processing by AuthenticateController#authenticate as HTML

2019-03-06T15:52:09.251+00:00 conjur-follower-2-9rg8m conjur-possum: I, [2019-03-06T15:52:09.012252 #900] INFO -- : Parameters: {"authenticator"=>"authn-k8s", "service_id"=>"ocp", "account"=>"<customer>", "id"=>"host/conjur/authn-k8s/ocp/apps/grp-inf-csi-conjur-ref/*

2019-03-06T15:52:09.251+00:00 conjur-follower-2-9rg8m conjur-possum: I, [2019-03-06T15:52:09.032582 #900] INFO -- : Completed 401 Unauthorized in 20ms

2019-03-06T15:52:09.251+00:00 conjur-follower-2-9rg8m nginx: 10.36.36.1 "POST /authn-k8s/ocp/<customer>/host%2Fconjur%2Fauthn-k8s%2Focp%2Fapps%2Fgrp-inf-csi-conjur-ref%2F%2A%2F%2A/authenticate HTTP/1.1" 401 5 "-" "Go-http-client/1.1" 0.022 0.023
```
    
I found this in the conjur-kubernetes-authenticator:0.11.1 sidecar log:
```
INFO: 2019/03/06 16:04:09 main.go:125: authenticating as host/conjur/authn-k8s/ocp/apps/grp-inf-csi-conjur-ref/*/* ...

INFO: 2019/03/06 16:04:09 requests.go:40: making authn request to https://urldefense.proofpoint.com/v2/url?u=https-3A__conjur-2Dfollower.grp-2Dinf-2Dcsi-2Dconjur.svc.cluster.local_authn-2Dk8s_ocp_<customer>_host-252Fconjur-252Fauthn-2Dk8s-252Focp-252Fapps-252Fgrp-2Dinf-2Dcsi-2Dconjur-2Dref-252F-252A-252F-252A_authenticate&d=DwIGaQ&c=E55fojPA83XrPGfndbiaQQ&r=AxSHpnf_u7R5KGUmB-VPZyxGaeV51dJ2XxlqDXdr21Y&m=bdR3gJ9kcUs7LpVdSrLBmhTJbXof82WY23KCxpWkFzk&s=XFu8s59Pc1rPInV9nY8HefTaDdOcVSw9HjLwePSkvY4&e=

INFO: 2019/03/06 16:04:09 main.go:109: waiting for 6 minutes to re-authenticate.

ERROR: 2019/03/06 16:04:09 main.go:82: on authenticate:
```
